crates/teloxide-core/src/types/input_file.rs

```diff
@@ -31,6 +31,7 @@ use crate::types::InputSticker;
 /// This object represents the contents of a file to be uploaded.
 ///
 /// [The official docs](https://core.telegram.org/bots/api#inputfile).
+#[serde_with_macros::skip_serializing_none]
```

Comment:
Structures that have a hand-written serialize impl can't use `skip_serializing_none`

Reply:
Fixed